src/api/routes/myGroups.ts

```diff
@@ -18,7 +18,7 @@ function isSubset<T>(superset: Set<T>, subset: Set<T>): boolean {
   return true;
 }
 
-function meetingLinkIsExpired(group: Group) {
+export function meetingLinkIsExpired(group: Group| {updatedAt: Date}) {
```

Comment:
Don't pass in `group`. Just pass in `updateAt`. And all your headache will be gone. #lawofdemeter

And don't call it `updatedAt`, because it would confuse the reader. Use `meetingLinkCreatedAt` or something like that.